packages/core/admin/package.json

```diff
@@ -133,7 +133,9 @@
     "yup": "^0.32.9"
   },
   "devDependencies": {
+    "@pmmmwh/react-refresh-webpack-plugin": "0.5.4",
```

Comment:
I think it should be set in the dependencies because we will also use it in generated apps when customising the admin panel or developing plugins.

WDYT?

Reply:
@soupette I agree. I push the changes.